cms/src/components/content/EditContentParentValidation.vue

```diff
@@ -69,6 +69,7 @@ const createTranslation = (language: LanguageDto) => {
         title: `Translation for ${language.name}`,
         slug: "",
         parentTags: [],
+        publishDateVisible: true,
```

Comment:
No need to set the `publishDateVisible` field on the `ContentDto` documents as this is done in the API.